tests/manual_tests/visual_screen_tests/visual_screen_tests.py

```diff
@@ -2,7 +2,7 @@
 
 import sys, os
 
-from asmcnc.comms.logging_system.logging_system import Logger
+from src.asmcnc.comms.logging_system.logging_system import Logger
```

Comment:
This won't work when running from easycut-smartbench

Reply:
Why not? I ran it from there in the terminal

Reply:
The src append happens later on